packages/cli/src/scanner/ai-scanner.ts

```diff
@@ -13,9 +11,7 @@ export class AIScanner {
     public async scan(url: string, chromePath?: string, sourcePath?: string): Promise<AxeScanResults> {
         try {
             console.log(`Starting accessibility scanning of URL ${url}.`);
-
             await this.page.create(chromePath);
-            await this.page.enableBypassCSP();
```

Comment:
Why do we no longer need this?

Reply:
Nevermind, it's because we do this anyway elsewhere in Page

Reply:
yes here https://github.com/microsoft/accessibility-insights-service/blob/d1fc305781244ad3b1819f2f45db02e01f6a9c80/packages/scanner-global-library/src/page-configurator.ts#L15